docs/getting_started.md

```diff
@@ -1,316 +1,46 @@
-
-```
-fondant==0.2.0
-imagesize==1.4.1
+fondant --help
 ```
 
-With our component complete we can now add it to our pipeline definition (`pipeline.py`):
+## Demo
+For demonstration purposes, we provide sample pipelines in the Fondant GitHub repository. A great starting point is the pipeline that loads and filters creative commons images. To follow along with the upcoming instructions, you can clone the [repository](https://github.com/ml6team/fondant) and navigate to the `examples/pipelines/filter-cc-25m` folder.
 
-```python
-extract_resolution = ComponentOp(
-    component_dir='components/extract_resolution', 
-)
+This pipeline loads an image dataset and reduces the dataset to png files. For more details on how you can build this pipeline from scratch, check out our [guide](/docs/guides/build_a_simple_pipeline.md). 
 
-my_pipeline.add_op(load_from_hf_hub) # this line was already there
-my_pipeline.add_op(extract_resolution, dependencies=load_from_hf_hub)
+## Running the sample pipeline and explore the data
+After navigating to the pipeline directory, we can run the pipeline by using the LocalRunner as follow:
 ```
-
-We add the component to our pipeline definition and specify that it depends on the `load_from_hf_hub` component. This means that the `load_from_hf_hub` component will be executed first and the output of that component will be passed to the `extract_resolution` component.
-
-We can now easily run our new pipeline:
-
-```bash
 fondant run pipeline --local
 ```
 
-You will see that the components runs sequentially and that each has its own logs. 
-
-Note that with custom components the image will be built as part of running the pipeline by leveraging a `build` spec in the docker-compose file. This means that you can change the code of your component and run the pipeline again without having to rebuild the image manually.
-
-
-We now have a simple pipeline that downloads a dataset from huggingface hub and extracts the width and height of the images. A possible next step is to create a component that [filters the data based on the aspect ratio](https://github.com/ml6team/fondant/tree/main/components/filter_image_resolution) ? Or run a [CLIP model on the images to get embeddings](https://github.com/ml6team/fondant/tree/main/components/embed_images)?
-
-## Explore the data
-
-
-Fondant includes a data explorer tools that lets you inspect the intermediate data of your pipeline. You can start the data explorer by running:
-
-```bash
-fondant explore --data-directory "path/to/your/data"
+The sample pipeline will run and execute three steps, which you can monitor in the logs. It will load data from HuggingFace, filter out images, and then download those images. The pipeline results will be saved to parquet files. If you wish to visually explore the results, you can use the data explorer.
+The following command will start the data explorer:
+```
+fondant explore --base_path ./data
```

Comment:
```suggestion
fondant explore --base_path <base_path_dir>
```

Reply:
the default base path of that pipeline is `./data` so I want to keer it that way to make the getting started as easy as possible

Reply:
makes sense, we can keep it then